src/mlpack/methods/random_forest/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(SOURCES
   bootstrap.hpp
   random_forest.hpp
   random_forest_impl.hpp
+  extra_trees_main.cpp
```

Comment:
This is not where to add this.  Take a look lower down---what you'll need to add the binding is this:

```
add_cli_executable(extra_trees)
add_python_binding(extra_trees)
add_julia_binding(extra_trees)
add_go_binding(extra_trees)
add_r_binding(extra_trees)
add_markdown_docs(extra_trees "cli;python;julia;go;r" "classification")
```

Then, CMake should try to build the binding correctly. :+1: